ATB.BL/Entities/FlightValidationResult.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ATB.Entities
+{
+    public class FlightValidationResult
+    {
+        public bool IsValid { get; set; }
```

Comment:
can we make this a derived property?

Reply:
namespace ATB.Entities
{
    public class FlightValidationResult
    {
        private List<string> _errors = new List<string>();

        public bool IsValid => _errors.Count == 0;

        public List<string> Errors => _errors;
    }
}